Earthfile

```diff
@@ -290,29 +295,41 @@ framework:
 
     SAVE ARTIFACT --keep-own /framework/ framework
 
+multi-build-framework-image:
+    ARG --required SECUIRTY_PROFILE
+
+    BUILD --platform=linux/amd64 --platform=linux/arm64 +build-framework-image
```

Comment:
you can do that?! :D

Reply:
yup, they do have some niiiice fetures :P